greenriver/hmis-frontend
# Make project dashboard default to enrollments

## Description

If the user has permission to view enrollments on a project, then navigating to the project should default to bringing up the enrollments page. Otherwise, if they don't have permission, then the current behavior (bringing up the overview page) should be maintained.
https://www.pivotaltracker.com/n/projects/2591838/stories/186877672

### Testing instructions
Here's the list that I used to help me test the different cases- I'll paste it here in case it's helpful to others testing and so you can make sure I haven't missed any:
- [ ] Clicking a project from the projects search
  - [ ] With permission to view enrollments on that project
  - [ ] Without permission to view enrollments on that project
- [ ] Clicking a project from an organization’s list of projects
  - [ ] With permission to view enrollments on that project
  - [ ] Without permission to view enrollments on that project
- [ ] Clicking a project from an enrollment
  - [ ] With permission to view enrollments on that project
  ~Without permission: I believe this cannot happen; if you're already viewing an enrollment on the project then you must have permission to view enrollments on that project~

## Type of change
[//]: # 'remove options that are not relevant'

## Checklist before requesting review
